api/src/test/java/org/openmrs/module/datafilter/impl/UserProgramBasedFilterTest.java

```diff
@@ -37,6 +41,14 @@ public class UserProgramBasedFilterTest extends BaseProgramBasedFilterTest {
 	@Autowired
 	private DataFilterService service;
 	
+	@Override
+	public Properties getRuntimeProperties() {
+		Properties props = super.getRuntimeProperties();
+		//Fixes the error reported by the h2 driver in tests
+		props.setProperty(Environment.URL, props.getProperty(Environment.URL) + ";DB_CLOSE_ON_EXIT=FALSE" + ";MODE=MYSQL");
```

Comment:
I think `DB_CLOSE_ON_EXIT=FALSE` is added in the superclass, can you verify that it is not getting added twice